### PR TITLE
fixes react-router error

### PR DESCRIPTION
**Pull request by ddaaggeett**
[this](https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]=) describes an error i ran into attempting to use react-router. fails on production build, so, this PR fixes that.

looks like the `BrowserRouter` had to be used per `client.js`, and `StaticRouter` per `server.js`.

can now:
`import { Link } from 'react-router-dom'`
< optional, obviously :) >

i recommend updating the project keywords to include webpack3 instead of 2.

### CODE REVIEW

File: source/js/client.js [hunk 1]
```diff
@@ -18,14 +19,16 @@ es6Promise.polyfill();
...
-      </Provider>
-    </AppContainer>,
-    document.getElementById('root')
-  );
+    ReactDOM.render(
+        <AppContainer>
+            <Provider store={ store }>
+                <BrowserRouter>
+                    <Component />
+                </BrowserRouter>
```
Comment (Stanko) [inline source/js/client.js]: I need to check how this affects HMR.

File: source/js/views/Server/index.jsx [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
 );
 
-Server.propTypes = {
-  location: PropTypes.string,
-  context: PropTypes.object,
-};
+// Server.propTypes = {
+//   location: PropTypes.string,
+//   context: PropTypes.object,
+// };
```
Comment (Stanko) [inline source/js/views/Server/index.jsx]: Remove if it isn't used anymore.

File: source/js/views/Client/index.jsx [hunk 3]
```diff
@@ -1,9 +1,6 @@
 import React from 'react';
-import { BrowserRouter } from 'react-router-dom';
 import App from 'views/App';
 
 export default () => (
-  <BrowserRouter>
     <App />
-  </BrowserRouter>
```
Comment (Stanko) [inline source/js/views/Client/index.jsx]: I think we can remove this view completely now and just load `App` instead of `Client`

**Comment by ddaaggeett** [comment]
made changes you requested. everything looks good. HMR seems to be working with `npm start`

**Comment by ddaaggeett** [comment]
sure thing, boss

**Pull request merged**